provisioning/database/database.bicep

```diff
@@ -33,9 +33,6 @@ resource privateDNSzoneForDatabase 'Microsoft.Network/privateDnsZones@2020-06-01
 }
 
 resource databaseServer 'Microsoft.DBforMySQL/flexibleServers@2021-05-01' = {
-  dependsOn: [
-    privateDNSzoneForDatabase
-  ]
```

Comment:
Did you successfully deploy without this? I know VS Code complains but I was experiencing issues actually getting it to deploy properly without it

Reply:
It wouldn't let me get past the database deploy portion with that at all

Reply:
Oh really it was throwing an error?

Reply:
Yea, it's too far up my terminal now, but you can see a warning in the file in vscode if you have it open

Reply:
Interesting - it it works without it though then perfect

Reply:
yea can verify after containers are up